qa/suites/rados/basic/tasks/rgw_snaps.yaml

```diff
@@ -8,8 +8,8 @@ overrides:
         osd_max_omap_entries_per_request: 10
 tasks:
 - rgw:
-    default_idle_timeout: 3600
-    client.0: null
+    client.0:
+      default_idle_timeout: 3600
```

Comment:
looks like i broke this in https://github.com/ceph/ceph/pull/15184 - `default_idle_timeout` was specific to fastcgi, so it's no longer being parsed and removed from `config`. this change should work, but it's probably better to remove the setting completely

it looks like the same issue is present in other upgrade tests:
> rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml:    default_idle_timeout: 300
upgrade/jewel-x/stress-split/7-final-workload/rgw-swift.yaml:    default_idle_timeout: 300
upgrade/kraken-x/stress-split/7-final-workload/rgw-swift.yaml    default_idle_timeout: 300

mind removing those too?